authority/voting/server/server.go

```diff
@@ -307,14 +364,47 @@ func New(cfg *config.Config) (*Server, error) {
 
 	// Start up the listeners.
 	for _, v := range s.cfg.Server.Addresses {
-		l, err := net.Listen("tcp", v)
-		if err != nil {
-			s.log.Errorf("Failed to start listener '%v': %v", v, err)
-			continue
+		// parse the Address line as a URL
+		u, err := url.Parse(v)
+		if err == nil {
+			switch u.Scheme {
```

Comment:
it looks like you're only getting `tcp://` here, not `tcp4://` and `tcp6://`

Reply:
yeah, I think we should get rid of tcp4 and tcp6 since there is no such thing ;-)

Reply:
updated to catch tcp4 and tcp6 as well.